core/src/main/java/hudson/model/Computer.java

```diff
@@ -827,23 +827,6 @@ protected void kill() {
         setNumExecutors(0);
     }
 
-    /**
-     * Called by {@link Jenkins#updateComputerList(boolean, Collection)} to notify {@link Computer} that it will be discarded.
-     *
-     * <p>
-     * Note that at this point {@link #getNode()} returns null.
-     *
-     * <p>
-     * Note that the Queue lock is already held when this method is called.
-     *
-     * @see #onRemoved()
-     */
-    @Restricted(NoExternalUse.class)
-    @GuardedBy("hudson.model.Queue.lock")
-    /*package*/ void inflictMortalWound() {
```

Comment:
Cute name, but unnecessary given `NoExternalUse`.

(Also why was it marked `@Restricted` when it was not `public`? This ought to be a compiler error.)